tests/unit/test_client.py

```diff
@@ -445,7 +445,7 @@ def test_client_signs_call(self, signer_mock):
             'request-created.myservice.test_operation', request=request,
             operation_name='test_operation')
 
-        signer_mock.return_value.sign.assert_called_with(
+        signer_mock.assert_called_with(
```

Comment:
FYI: This pre-existing test case is adjusted a little bit to reflect the latest code structure, but the observable behavior of sign() is NOT changed, so this should not be considered as a breaking change.